### DISCUSSION TITLE
使用toolkit评测UAV123的问题

### DISCUSSION THREAD

**Discussion by Janghhy** [STVIR/pysot#360] [General] [answered]
想要在评测UAV123数据集时去除目标不在图像中的帧，即标记为NaN的帧，应该怎么办？

**Comment by ZhiyuanChen** [comment]
我无法理解你的问题。
请详细指出你在哪里遇到了问题？预期的结果是什么？实际的结果是什么？

**Comment by Janghhy** [comment]
感谢您的回复，可以使用您发的文件正常评估。但是UAV123这个数据集中存在一些目标不在视图中的情况。我想问的就是，如何把这些目标不在视图中的帧去掉然后进行评估？（比如要对json文件和跑出来的txt文件做些什么）

**Comment by ZhiyuanChen** [comment, marked as answer]
这取决于你想做些什么。
具体来说，你可以在JSON中把所有目标不在的帧删掉，可以在评估的时候不对这样的帧进行评估，还可以在最终结果中去掉这样的帧。
